src/wp-content/themes/twentyfifteen/js/skip-link-focus-fix.js

```diff
@@ -1,21 +1,27 @@
 /**
- * Makes "skip to content" link work correctly in IE9, Chrome, and Opera
```

Comment:
IMO, if we aren't using this code anymore, should we be changing it? _If it ain't broke?_ 

I think I'd rather not touch it if we're on the way to removing it altogether. 

From the PR description and https://github.com/sabernhardt/wordpress-develop/pull/22#discussion_r1200858259 I understand you simply updated this Twenty Fifteen code from the code that is in Twenty Seventeen. So in that case, not really a big deal.

Reply:
Twenty Fifteen and Twenty Sixteen are broken on iPhone due to the script running on WebKit: https://core.trac.wordpress.org/ticket/38338#comment:30

Reply:
Ah, thanks for that context. Could you update the PR description to note that?